### ISSUE TITLE
Control Panel contrast & accessibility

### ISSUE THREAD

**Issue by daltonrooney** [craftcms/cms#5275]
I know a lot of improvements were made to the CP in Craft 3 and I believe there are more on the way for Craft 4. I'd just like to put in a vote for more contrast in the admin, especially things like the Matrix title bars, text fields, and buttons. Many of the clients we've trained on Craft have mentioned the lack of contrast and I believe it does make the CP more difficult to use.

Here's a quick before & after of an entry screen we're working on for a client:

![default CP](https://user-images.githubusercontent.com/742045/69754679-64b33500-110b-11ea-956e-22c8984c2a1e.png)
Default CP

![modified CP](https://user-images.githubusercontent.com/742045/69754687-6846bc00-110b-11ea-953f-a631a0891c53.png)
Modified CP

We're using Matrixmate here so a couple of the elements are not part of the standard Craft CP, but we've received good feedback on the changes to the Matrix title bar, and the subtle incease in contrast on the borders of buttons and text fields.

Sure this could be done as a plugin, but I think the default CP experience should be advancing accessibility with every version.

**Comment by daltonrooney** [comment]
Update: Just found [this very active closed issue](https://github.com/craftcms/cms/issues/2883) regarding CP & accessibility. Looks like some good improvements are being made for 3.4. Maybe I will open some more specific tickets after I've had a chance to install 3.4 and take it for a spin.

**Comment by brandonkelly** [comment]
Thanks! 

To update to the 3.4 branch, change your `craftcms/cms` requirement in composer.json to:

```json
"require": {
  "craftcms/cms": "3.4.x-dev",
  "...": "..."
}
```

Then run `composer update`.
